src/data/services/LmsApiService.js

```diff
@@ -137,7 +137,7 @@ class LmsApiService {
   }
 
   static updateCanvasConfig(formData, id) {
-    return apiClient.patch(`${LmsApiService.lmsIntegrationUrl}/canvas/configuration/${id}/`, formData, 'json');
+    return apiClient.put(`${LmsApiService.lmsIntegrationUrl}/canvas/configuration/${id}/`, formData, 'json');
```

Comment:
is the formData here always going to replace the entire config, I had thought the patch previously was about only updating some of the fields,,, if this has the desired effect in the backend, we are all set!

Reply:
The way it is setup it will always send the full form data anyways.
The reason I swapped this - and apologies, I meant to comment on this line - is because of these 2 things combining together:
- The form component is storing the value as "null" instead of "false"
- sending a null value via the patch endpoint says "don't change this" rather than "this is changing to false"

Reply:
got it, makes sense now, just checking on accidental overwriting/losing of existing data, since that won't happen we are good, thanks!

Reply:
no worries. I meant to explain this since it was definitely a weird change.